airflow/config/airflow.cfg

```diff
@@ -0,0 +1,214 @@
+[core]
```

Comment:
This is a fine default, obviously we will want to override this in the chart.  Will comment there on how.